pylint/reporters/text.py

```diff
@@ -199,11 +203,9 @@ def write_message(self, msg: Message) -> None:
     def handle_message(self, msg: Message) -> None:
         """Manage message of different type and in the context of path."""
         if msg.module not in self._modules:
-            if msg.module:
-                self.writeln(f"************* Module {msg.module}")
-                self._modules.add(msg.module)
-            else:
-                self.writeln("************* ")
```

Comment:
I removed the `if msg.module` check here and below because I cannot see when it would ever be used? It's also not tested in coverage. If you think we need more safety though, I can add it above and turn

`if msg.module not in self._modules:`

to

 `if msg.module and msg.module not in self._modules:` which seems reasonable too